front-end-admin/src/components/callout/CalloutInfo.vue

```diff
@@ -1,54 +1,86 @@
 <template>
   <div class="jumbotron">
     <div class="container">
-      <h2>{{callout.title}}</h2>
-      <!-- TODO: List out associated needs with a specific callout -->
-      <h2>Update Callout Information</h2>
-      <form v-on:submit.prevent="onFormSubmit">
-        <div class="form-group">
-          <label class="control-label col-sm-2">Title:</label>
-          <input v-model="callout.name" type="text" width="50" required>
+      <br>
+      <br>
+      <b-card title="Edit Callout">
+      <b-card :title="callout.name">
+        <b-form @submit="onFormSubmit" class="form-horizontal" >
+          <input type="hidden" name="callout_id" v-model="callout.id">
+          <h5>Callout Name: </h5>
+          <b-form-textarea v-model="callout.name"
+                           required
+                           placeholder="Enter name.."
+                           :rows="1"
+                           no-resize
+                           name="name"
+                           :max-rows="1">
+          </b-form-textarea>
+          <h5>Description: </h5>
+          <b-form-textarea v-model="callout.descriptionMessage"
+                           required
+                           placeholder="Enter description.."
+                           :rows="3"
+                           name="Amazon URL"
+                           :max-rows="3">
+          </b-form-textarea>
+          <h5>Set End Date: </h5>
+          <br>
+          <b-button type="submit" variant="primary">Submit</b-button>
+        </b-form>
+      </b-card>
+        <b-card>
+      <div v-show="calloutNeeds.length > 0">
```

Comment:
You'll want to use v-if here. v-show is meant for conditional elements that can be turned on and off often. v-if has a better performance cost since it won't even render this markup unless there is a need. For more info, the Vue documentation has a section on "v-if vs v-show" on this page: https://vuejs.org/v2/guide/conditional.html

Reply:
Ohh very cool. Changed it!